src/main/java/net/sf/jabref/JabRefPreferences.java

```diff
@@ -862,14 +860,14 @@ private JabRefPreferences() {
         defaults.put(USE_CONVERT_TO_EQUATION, Boolean.FALSE);
         defaults.put(USE_CASE_KEEPER_ON_SEARCH, Boolean.TRUE);
         defaults.put(USE_UNIT_FORMATTER_ON_SEARCH, Boolean.TRUE);
+    }
 
+    public String getUser() {
         try {
-            defaults.put(USER_FILE_DIR_IND_LEGACY, Globals.FILE_FIELD + Globals.DIR_SUFFIX + '-' + get(DEFAULT_OWNER) + '@' + InetAddress.getLocalHost().getHostName()); // Legacy setting name - was a bug: @ not allowed inside BibTeX comment text. Retained for backward comp.
-            defaults.put(USER_FILE_DIR_INDIVIDUAL, Globals.FILE_FIELD + Globals.DIR_SUFFIX + '-' + get(DEFAULT_OWNER) + '-' + InetAddress.getLocalHost().getHostName()); // Valid setting name
+            return get(DEFAULT_OWNER) + '-' + InetAddress.getLocalHost().getHostName();
```

Comment:
̀USER_FILE_DIR_INDIVIDUAL` was never a real preference and only used as a key to some other preference, i.e. the code read `pref.get(pref.get(USER_FILE_DIR_INDIVIDUAL))` before and now takes the form `pref.get(pref.getUser())`. Moreover, `USER_FILE_DIR_INDIVIDUAL` was never set and so stayed at the default value.